src/viam/sdk/services/mlmodel/private/proto.cpp

```diff
@@ -230,16 +286,13 @@ class tensor_to_pb_value_visitor : public boost::static_visitor<::grpc::Status>
                 // Base64 encode the entire last dimension stride into a string.
                 const auto* const bytes_begin = &tensor.element(ixes.begin(), ixes.end());
                 ixes.back() = tensor.shape().back();
-                const auto* const bytes_end = bytes_begin + ixes.back();
-                const google::protobuf::StringPiece bytes{
-                    reinterpret_cast<const char*>(bytes_begin),
-                    std::size_t(bytes_end - bytes_begin)};
                 std::string encoded;
-                google::protobuf::Base64Escape(bytes, &encoded);
+                absl::Base64Escape({reinterpret_cast<const char*>(bytes_begin), ixes.back()},
```

Comment:
More switching to absl, and the block removed above was actually largely unnecessary, since we can just pass the size directly into the `StringPiece` / `string_view` constructor, rather than subtracting the pointers, one of which was just an advance of the other.